.env.template

```
@@ -45,3 +45,10 @@ ACTIVE_CONDITIONS_ONLY=false
 AUTHENTICATED=false
 CERT_PATH_PUBLIC=
 CORS_POLICY=demo
+
+#JETSTREAM
+STARTUP_TYPE=jetstream
+SERVER_URL=0.0.0.0:14222
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SERVER_URL key should go before the STARTUP_TYPE key

(UnorderedKey)